### PR TITLE
New version: Azul.Zulu.17.JDK version 17.48.15

### LINKED ISSUE #137658: [Update Request]: Azul Zulu 17 JDK
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Azul.Zulu.17.JDK

### Package Version

17.48.15

### Please describe the changes you would like to see

There is a new version released. Download link: https://www.azul.com/downloads/?os=windows#zulu

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
- Resolve: #137658

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137674)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   EF574C4C60DBE2B0E4FF6D4AFAF5D82AF47E0D98050D3CA283A74C500369F381
 Expected hash: fea202e2dfd5e116719993effc800b3cae1d0fdbe301982d823f44f88551c2b0

(Automated response - build 695.)

File: manifests/a/Azul/Zulu/17/JDK/17.48.15/Azul.Zulu.17.JDK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - DisplayName: Azul Zulu JDK 17.44.15 (17.0.8), 64-bit
+    UpgradeCode: '{0EFBE51F-F003-4A7D-8CFF-2055F4F401ED}'
+- Architecture: arm64
+  InstallerType: portable
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: zulu17.48.15-ca-jdk17.0.10-win_aarch64\bin\java.exe
+    PortableCommandAlias: java
+  InstallerUrl: https://cdn.azul.com/zulu/bin/zulu17.48.15-ca-jdk17.0.10-win_aarch64.zip
+  InstallerSha256: FEA202E2DFD5E116719993EFFC800B3CAE1D0FDBE301982D823F44F88551C2B0
```
Comment (stephengillie) [inline manifests/a/Azul/Zulu/17/JDK/17.48.15/Azul.Zulu.17.JDK.installer.yaml:45]:
```suggestion

```

(Automated response - build 695.)

Comment (stephengillie) [inline manifests/a/Azul/Zulu/17/JDK/17.48.15/Azul.Zulu.17.JDK.installer.yaml:45]:
```suggestion
  InstallerSha256: EF574C4C60DBE2B0E4FF6D4AFAF5D82AF47E0D98050D3CA283A74C500369F381
```

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by stephengillie — APPROVED** [review]

**Pull request merged**